actors.md

```diff
@@ -38,7 +74,7 @@ func CreateStorageMiner(pubkey PublicKey, pledge BytesAmount, pid PeerID) Addres
         Fatal("not enough funds to cover required collateral")
     }
     
-    newminer := VM.CreateActor(MinerActor, msg.Value, pubkey, pledge, pid)
+    newminer := InitActor.Exec(MinerActorCodeCid, EncodeParams(pubkey, pledge, pid))
```

Comment:
How about retaining the descriptive `InitActor.CreateActor` name? `Exec` is opaque and also limits future change.

Reply:
We really wanted to stick to unix terminology here, where `exec` is the accepted terminology for taking a given piece of code and instantiating a new process running it.